src/app/store/accounts/account.models.ts

```diff
@@ -1,24 +1,24 @@
-import { Account } from '@stacks/wallet-sdk';
+import type { Account } from '@stacks/wallet-sdk';
 
 import type { AccountBalanceStxKeys } from '@shared/models/account.model';
 
 // Extending the `Account` type from `@stacks/wallet-sdk`
-export type SoftwareWalletAccountWithAddress = Account & {
+export type SoftwareWalletAccount = Account & {
   type: 'software';
   address: string;
   stxPublicKey: string;
   dataPublicKey: string;
 };
 
-export interface LedgerAccountWithAddress {
+export interface LedgerWalletAccount {
   type: 'ledger';
```

Comment:
`HardwareWalletAccount` with type `ledger`?

Reply:
good suggestion, renamed. Left properties as they are, but were we to support other hw wallets, it'd probably have to be something like type = hardware, model = ledger or something.